pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -118,6 +118,9 @@ type SchedulerVolumeBinder interface {
 	// This function is called serially.
 	AssumePodVolumes(assumedPod *v1.Pod, nodeName string) (allFullyBound bool, err error)
 
+	// RevertAssumedPodVolumes will revert assumed PV and assumed PVC in cache.
```

Comment:
nit: `s/will revert assumed PV and assumed PVC in cache/reverts assumed PV and PVC cache`